### DISCUSSION TITLE
error with installing chainlink

### DISCUSSION THREAD

**Discussion by HussamX98** [Cyfrin/foundry-full-course-cu#1368] [Q&A]
so I've been trying to run the $ forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit however for some reason it is returning this:  Installing chainlink-brownie-contracts in C:/Users/alsal/foundry-simple-storage-f23/foundry-fund-me-f23/foundry-smart-contract-lottery-f23/foundry-erc20-f23/foundry-nft-f23/foundry-defi-stablecoin-f23/lib\chainlink-brownie-contracts (url: Some("https://github.com/smartcontractkit/chainlink-brownie-contracts"), tag: Some("0.6.1"))
Cloning into 'C:/Users/alsal/foundry-simple-storage-f23/foundry-fund-me-f23/foundry-smart-contract-lottery-f23/foundry-erc20-f23/foundry-nft-f23/foundry-defi-stablecoin-f23/lib/chainlink-brownie-contracts'...
remote: Enumerating objects: 32923, done.
remote: Counting objects: 100% (2524/2524), done.
remote: Compressing objects: 100% (1050/1050), done.
remote: Total 32923 (delta 1447), reused 2336 (delta 1271), pack-reused 30399
Receiving objects: 100% (32923/32923), 46.85 MiB | 10.10 MiB/s, done.
Resolving deltas: 100% (8658/8658), done.
fatal: cannot write keep file 'C:/Users/alsal/foundry-simple-storage-f23/foundry-fund-me-f23/foundry-smart-contract-lottery-f23/foundry-erc20-f23/foundry-nft-f23/foundry-defi-stablecoin-f23/.git/modules/lib/chainlink-brownie-contracts/objects/pack/pack-138a398f0250df91bd09a3a22ad26265178d1cfd.keep': Filename too longfatal: fetch-pack: invalid index-pack output
fatal: clone of 'https://github.com/smartcontractkit/chainlink-brownie-contracts' into submodule path 'C:/Users/alsal/foundry-simple-storage-f23/foundry-fund-me-f23/foundry-smart-contract-lottery-f23/foundry-erc20-f23/foundry-nft-f23/foundry-defi-stablecoin-f23/lib/chainlink-brownie-contracts' failed
Error: 
git submodule exited with code 128 
I've tried using git init, restarting computer, shutting down vscode but nothing worked does anyone know how to fix this issue?

**Comment by tobezhanabi** [comment]
if you face submodel 128 error do this
 switch to faster network and then 
git config --global http.postBuffer 524288000 and then
git gc --aggressive

**Reply by HussamX98** [reply]
it says nothing new to pack

**Reply by tobezhanabi** [reply]
nothing new to pack? did you switch to a faster network

**Comment by heheboii11** [comment]
keep on trying and have a good internet connection

**Comment by owanemi** [comment]
i have the same problem too, you could just download the zip file and use it locally

**Reply by EngrPips** [reply]
Interesting, Thanks for the heads-up.
